lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -396,8 +419,7 @@ async sub _set_transaction_type {
     my $accounts = await $self->accounts;
     return undef unless $accounts;
 
-    my @accounts_response = $accounts->@*;
-    my %accounts = map { lc($_) => 1 } @accounts_response;
+    my %accounts = $accounts->%*;
```

Comment:
probably no real reason to de-reference this just use it in its reference form below